packages/webview_flutter/lib/src/tizen_webview_controller.dart

```diff
@@ -158,6 +162,17 @@ class TizenWebViewController extends PlatformWebViewController {
   @override
   Future<void> setUserAgent(String? userAgent) =>
       _webview.setUserAgent(userAgent);
+
+  @override
+  Future<void> setOnPlatformPermissionRequest(
+    void Function(
+      PlatformWebViewPermissionRequest request,
+    ) onPermissionRequest,
+  ) async {
+    throw UnimplementedError(
+        'This version of `TizenWebViewController` currently has no '
+        'implementation.');
+  }
```

Comment:
What additional features are needed in the platform to support the setOnPlatformPermissionRequest interface?
Could this be supported in the future?